graph/src/env.rs

```diff
@@ -850,6 +856,12 @@ struct Inner {
     graphql_max_skip: u32,
     #[envconfig(from = "GRAPHQL_ALLOW_DEPLOYMENT_CHANGE", default = "false")]
     graphql_allow_deployment_change: EnvVarBoolean,
+    // usize::MAX
+    #[envconfig(
+        from = "GRAPH_GRAPHQL_WARN_RESULT_SIZE",
+        default = "18446744073709551615"
```

Comment:
Not sure if we could patch it, but it would be nice if `envconfig` accepted some kind of computation and/or value such as `usize::MAX`.

Reply:
This is fixed already in the latests commits. We're using const generics as a workaround, see `graph::env::WithDefaultUsize`.